package.json

```diff
@@ -40,7 +40,9 @@
       "plugin:vue/essential",
       "eslint:recommended"
     ],
-    "rules": {},
+    "rules": {
```

Comment:
If we imagine this was going to go to production, we wouldn't want this rule off. We shouldn't have console log statements in live code.

Reply:
Ok, good to know. I've removed it.